synapsis/src/main/java/com/soen/synapsis/appuser/connection/ConnectionService.java

```diff
@@ -1,15 +1,49 @@
 package com.soen.synapsis.appuser.connection;
 
 import com.soen.synapsis.appuser.AppUser;
+import com.soen.synapsis.appuser.AppUserRepository;
 import com.soen.synapsis.appuser.Role;
 import org.springframework.stereotype.Service;
 
+import java.util.ArrayList;
+import java.util.List;
+
 @Service
 public class ConnectionService {
     private final ConnectionRepository connectionRepository;
+    private final AppUserRepository appUserRepository;
 
-    public ConnectionService(ConnectionRepository connectionRepository) {
+    public ConnectionService(ConnectionRepository connectionRepository, AppUserRepository appUserRepository) {
         this.connectionRepository = connectionRepository;
+        this.appUserRepository = appUserRepository;
+    }
+
+    public List<AppUser> getConnections(Long user_uid) {
```

Comment:
I would suggest using the AppUser as a parameter rather than the id since we are using an object-oriented language

Reply:
Changes done